sys-apps/pick/pick-4.0.0-r1.ebuild

```diff
@@ -19,11 +19,12 @@ BDEPEND="virtual/pkgconfig"
 
 PATCHES=(
 	"${FILESDIR}/${PN}-4.0.0-tinfo.patch"
+	"${FILESDIR}/${PN}-4.0.0-fix-build-for-clang16.patch"
 )
 
 src_configure() {
 	# not autoconf
```

Comment:
For non-autoconf, we shouldn't use `econf` because it might handle future changes to `econf` incorrectly or mishandle passed arguments.